### DISCUSSION TITLE
How to remove "source string" Glossary terms ?

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#9920] [Q&A] [answered]
See this for example

https://translate.codeberg.org/browse/backintime/glossary/en/?q=&sort_by=target

I'm not admin of that Weblate instance but admin of that project. I would like to remove some of the source string terms from the Glossary. But I don't see a remove string button or menu entry somewhere.

I'm only able to remove Glossary terms from other languages.

**Comment by Geeyun-JY3** [comment]
You should open the source string you want to remove in the full editor. There is a <kbd>Tools</kbd> menu:
![image](https://github.com/WeblateOrg/weblate/assets/49396421/0cc5441e-7357-4b8a-985b-f96175951de4)

**Comment by buhtz** [comment]
This menu entry is missing in my instance.

![image](https://github.com/WeblateOrg/weblate/assets/11861496/134f08ac-5caa-428a-958f-51f0fd375939)

It is Weblate 5.0.1

**Reply by Geeyun-JY3** [reply]
Does the admin of Codeberg Translate or this project do something to the [default teams and roles](https://docs.weblate.org/en/weblate-5.0.1/admin/access.html#default-teams-and-roles)? Does your team and role have the relevant permission to remove glossary entry?

**Reply by buhtz** [reply]
Thanks for your reply. I opened [a ticket at Codeberg](https://codeberg.org/Codeberg/Community/issues/1217).

**Reply by Geeyun-JY3** [reply]
Emm actually you can go to `Setting` > `Teams` (For example https://hosted.weblate.org/accounts/profile/#teams) to see your teams and roles. Maybe there are some clues.

**Reply by buhtz** [reply]
![image](https://github.com/WeblateOrg/weblate/assets/11861496/56f33e38-27b8-4c3f-96bd-141b92f6efa2)

**Reply by Geeyun-JY3** [reply]
According to the [documentation](https://docs.weblate.org/en/weblate-5.0.1/admin/access.html#list-of-privileges-and-built-in-roles), the default `Administration` role has the `Delete glossary entry` permission. 
Can you see the permission definition of that project’s `Administration` role?

**Reply by buhtz** [reply]
No, I'm not able to see that. We have to wait for the admins of that instance. They are informed.

**Reply by buhtz** [reply]
Funny. Now I'm not able to delete any of the glossary strings. Not the source strings not the translated strings.
I'm still asking my servers admins.

**Reply by buhtz** [reply, marked as answer]
Got it.
In the components "Settings" -> "Translation" is a checkbox "Manage strings".

It was disabled.

**Reply by nijel** [reply]
The “Manage strings” is turned on by default for glossaries:

https://github.com/WeblateOrg/weblate/blob/959362b85f9f09cbe1d2ec1130c1307c22120f44/weblate/trans/models/project.py#L539

**Comment by buhtz** [comment]
Then take this as a bug report. Your UI is very hard to read and understand.

**Reply by nijel** [reply]
Can you please clarify what part you find hard to read and understand?
